<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeRangeInputWidget</class>
 <widget class="QWidget" name="TimeRangeInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Range</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QTimeEdit" name="m_startTimeOfDayInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>hh:mm:ss AP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="m_startTimeLabel">
       <property name="text">
        <string>Start Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="m_startTimeOffsetButton">
       <property name="text">
        <string>Time Offset</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">m_startTimeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="m_startTimeOffsetLayout" stretch="1,1,0">
       <item>
        <widget class="QSpinBox" name="m_startTimeInputSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_startTimeUnitsComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>minutes</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_startTimeSpecifierLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ago</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="m_startTimeOfDayButton">
       <property name="text">
        <string>Time of Day</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">m_startTimeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="m_timeSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="4">
      <widget class="QTimeEdit" name="m_endTimeOfDayInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>hh:mm:ss AP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="m_endTimeOffsetButton">
       <property name="text">
        <string>Time Offset</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">m_endTimeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="m_endTimeOfDayButton">
       <property name="text">
        <string>Time of Day</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">m_endTimeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="4">
      <layout class="QHBoxLayout" name="m_endTimeOffsetLayout" stretch="1,1,0">
       <item>
        <widget class="QSpinBox" name="m_endTimeInputSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_endTimeUnitsComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>minutes</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_endTimeSpecifierLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ago</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="m_endTimeLabel">
       <property name="text">
        <string>End Time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QRadioButton" name="m_realTimeButton">
       <property name="text">
        <string>Real Time</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">m_endTimeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="m_presentButton">
       <property name="text">
        <string>Present</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">m_startTimeButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="m_startTimeButtonGroup"/>
  <buttongroup name="m_endTimeButtonGroup"/>
 </buttongroups>
</ui>
